board.ts

```diff
@@ -23,26 +26,28 @@ export class Cell {
     this.container.y = y * cellHeight
 
     this.sprites = {
-      [SUNNY] : this.drawCell(0xc4f2fc),
-      [CLOUDS]: this.drawCell(0xcccccc),
-      [RAIN]: this.drawCell(0x635d60),
+      [WeatherType.SUNNY] : this.drawCell(0xc4f2fc),
+      [WeatherType.CLOUDS]: this.drawCell(0xcccccc),
+      [WeatherType.RAIN]: this.drawCell(0x635d60),
+      [WeatherType.HAIL]: this.drawCell(0xdddddd),
```

Comment:
The ts compiler required that we define mappings for each `WeatherType` here. Pretty cool, I think.